docs/getting_started/index.md

```diff
@@ -14,4 +14,3 @@ The following docs are a brief introduction to SDK development.
 -   [Running your first example](./first_example.md)
 -   [SDK Basics](./SDKBasics.md)
 -   [ZAP](./zap.md)
```

Comment:
So I noticed this across all the different categories... the menu rendered with Sphinx can show a different order than this manual one, and the actual page title in the menu doesn't always match this link text.

Is it a Sphinx thing to require these top level pages that are just a list of links?  (I'm not that familiar with Sphinx, but in the dev docs I manage we'd never have pages like this).

Reply:
I honestly don't know - this wasn't a change i made in this PR, just removing a now dead link. I'll have to defer to you and Sam for getting the organization to appear correct, I'm just trying to get the content in roughly the right spots at this point.

Reply:
Thanks, I'll check with Sam then.